src/h2olog/generated_raw_tracer.cc

```diff
@@ -95,8 +95,106 @@ static std::string gen_bpf_header() {
 }
 
 
+enum event_id_t {
+  T_SCHED_SCHED_PROCESS_EXIT,
```

Comment:
What does T stand for?

As this definition is to be used from both C and C++, my understanding is that it is better to use a common prefix for all the constants being defined to minimize the chance of collisions. But I do not see how we would come up with T - it does not seem to represent a particular "event_id."

Reply:
That's a great point. Totally agreed. In addition, it never makes sense if we use short prefixes for generated code, so I've changed it to use the longest prefixes as f94f80c38.